ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -166,93 +157,112 @@ export const AnacondaWizard = ({ dispatch, isBootIso, osRelease, storageData, lo
         return stepIdx <= currentStepIdx;
     };
 
+    const componentProps = {
+        setIsFormValid,
+        onCritFail,
+        onAddErrorNotification,
+        stepNotification,
+        isInProgress,
+        storageEncryption,
+        setStorageEncryption,
+        storageScenarioId,
+        isBootIso,
+        osRelease,
+        setStorageScenarioId: (scenarioId) => {
+            window.sessionStorage.setItem("storage-scenario-id", scenarioId);
+            setStorageScenarioId(scenarioId);
+        },
+    };
+
     const createSteps = (stepsOrder) => {
-        const steps = stepsOrder.filter(s => !s.isHidden).map(s => {
-            let step = ({
-                id: s.id,
+        return stepsOrder.filter(s => !s.isHidden).map(s => {
+            let stepProps = {
+                ariaLabel: `${title} content`,
                 name: s.label,
+                id: s.id,
                 stepNavItemProps: { id: s.id },
+                ...(s.isExpandable && { isExpandable: true }),
                 canJumpTo: canJumpToStep(s.id, currentStepId),
```

Comment:
canJumpTo seems to be removed in PF5, I guess we'll need to use isDisabled instead and the logic used in `canJumpToStep` will need to be modified a bit.